.gitignore

```diff
@@ -0,0 +1 @@
+.idea
```

Comment:
Please remove this file.

Reply:
I removed it. Is there any policy not to have .gitignore in this repository.

Reply:
Thanks

If there are things that get constructed by tests that should not be committed then we use them (this happens in plot), otherwise no. They would collect all the various editer and build environment cruft from contributors and we don't want that.

Reply:
That make sense.